src/resolve.ts

```diff
@@ -66,6 +66,12 @@ export type ResolveOptions = {
    * the resolver returns `undefined` instead of throwing an error.
    */
   try?: boolean;
+
+  /**
+   * Keep symlinks instead of resolving them.
+   * Default behavior is to resolve symlinks to their real paths.
+   */
+  preserveSymlinks?: boolean;
```

Comment:
if this is default node behavior we can drop flag (because otherwise it should be also supported in other logic without fast path to avoid following symlinks)